tests/vufind.php-cs-fixer.php

```diff
@@ -19,12 +19,18 @@
     'concat_space' => ['spacing' => 'one'],
     'ereg_to_preg' => true,
     'get_class_to_class_keyword' => true,
+    'global_namespace_import' => [
+        'import_constants' => null,
+        'import_functions' => true,
+        'import_classes' => null,
+    ],
     'is_null' => true,
     'linebreak_after_opening_tag' => true,
     'lowercase_cast' => true,
     'magic_constant_casing' => true,
     'modernize_strpos' => true,
     'native_function_casing' => true,
+    'native_function_invocation' => true,
```

Comment:
Would it make sense to do this only for the optimizable functions? See https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v2.15.1/src/RuleSet.php#L194 and https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/4448.

Reply:
Good idea, ill add and test that also